easybuild/easyconfigs/p/parallel/parallel-20171022-intel-2017b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'parallel'
+version = '20171022'
+
+homepage = 'http://savannah.gnu.org/projects/parallel/'
+description = """parallel: Build and execute shell commands in parallel"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
```

Comment:
This is more something for `GCCcore`?

Reply:
Well, maybe, but it requires Perl, so that would load the `GCCcore` Perl that we have to use as a build dep, making the `parallel` module incompatible with a full `Perl` installation done with `intel` toolchain...

Reply:
I still think it's a bad choice to put this under a full blown toolchain. It's not performance related and requires only a small part of Perl.

Reply:
What other option do we have if you want to be able to combine this with a full Perl that's built with `intel`?